src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1

```diff
@@ -184,23 +188,7 @@ function Test-CreateAndGetLogicAppUsingDefinitionWithActions
 	$workflow3 = Get-AzLogicApp -ResourceGroupName $resourceGroupName -Name $workflowName -Version $workflow1.Version
 	Assert-NotNull $workflow3
 
-	# Test 4: Get all workflows in ResourceGroup
```

Comment:
Why are you removing these three tests?

Reply:
That was a copy paste mistake. Will add back in